src/core/packages/saved-objects/api-server-internal/src/lib/apis/bulk_create.ts

```diff
@@ -201,122 +205,135 @@ export const performBulkCreate = async <T>(
     objects: authObjects,
   });
 
+  const inaccessibleObjects = authorizationResult?.inaccessibleObjects
+    ? Array.from(authorizationResult.inaccessibleObjects)
+    : [];
+
+  const expectredAuthorizedResults =
```

Comment:
typo: 
```suggestion
  const expectedAuthorizedResults =
```

Reply:
Addressed in 2c2c63c6c53343c5f2be1d72bfd6f143d71a030e